src/GitcoinGovernor.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+// forgefmt: disable-start
+import { GovernorCountingSimple, Governor } from "openzeppelin-contracts/governance/extensions/GovernorCountingSimple.sol";
+import { GovernorVotesComp, ERC20VotesComp } from "openzeppelin-contracts/governance/extensions/GovernorVotesComp.sol";
+import { GovernorTimelockCompound, ICompoundTimelock } from "openzeppelin-contracts/governance/extensions/GovernorTimelockCompound.sol";
+import { GovernorSettings } from "openzeppelin-contracts/governance/extensions/GovernorSettings.sol";
+// forgefmt: disable-end
+
+/**
+ * @notice The upgraded Gitcoin Governor—Bravo compatible and built from OpenZeppelin.
+ */
+contract GitcoinGovernor is
+  GovernorCountingSimple,
+  GovernorVotesComp,
+  GovernorTimelockCompound,
+  GovernorSettings
+{
+  /// @notice The address of the existing GTC token on Ethereum mainnet which will interop with
+  /// this Governor after the upgrade is complete.
+  ERC20VotesComp private constant GTC_TOKEN =
+    ERC20VotesComp(0xDe30da39c46104798bB5aA3fe8B9e0e1F348163F);
+
+  /// @notice The address of the existing GitcoinDAO Timelock on Ethereum mainnet which this
+  /// Governor will control after the upgrade is complete.
+  ICompoundTimelock private constant TIMELOCK =
+    ICompoundTimelock(payable(0x57a8865cfB1eCEf7253c27da6B4BC3dAEE5Be518));
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "GTC Governor Bravo";
+
+  /// @notice The number of GTC (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  uint256 private constant QUORUM = 2_500_000_000_000_000_000_000_000; // 2,500,000 GTC
+
+  /**
+   * @param _initialVotingDelay The deployment value for the voting delay this Governor will
+   * enforce.
+   * @param _initialVotingPeriod The deployment value for the voting period this Governor will
+   * enforce.
+   * @param _initialProposalThreshold The deployment value for the number of GTC required to submit
+   * a proposal this Governor will enforce.
+   */
+  constructor(
+    uint256 _initialVotingDelay,
+    uint256 _initialVotingPeriod,
+    uint256 _initialProposalThreshold
+  )
+    GovernorVotesComp(GTC_TOKEN)
+    GovernorSettings(_initialVotingDelay, _initialVotingPeriod, _initialProposalThreshold)
+    GovernorTimelockCompound(TIMELOCK)
```

Comment:
Does this imply the new OZ governor will be compatible with the existing timelock, and confirms that a new one is not needed? Seems like yes from the code

Reply:
Yes, as far as I can tell that is correct. We will confirm this 100% via simulation and testing in future PRs.